### ISSUE TITLE
Messy Generate Password UI in 6.4beta2

### ISSUE THREAD

**Issue by jacobshare** [jfarthing84/theme-my-login#14]
Thanks for this beta but I'm sure users are going to get confused because it looks like they need to copy the generated (strong) password and paste it into the empty password field that initially appeared beside the Generate Password button. 

Before clicking Generate Password (including empty field):
https://www.dropbox.com/s/evqzdfoy01f3nkj/before%20clicking%20Generate%20Password.png?dl=0

After clicking Generate Password (messy):

https://www.dropbox.com/s/8sjmvrlhyha7k7c/after%20clicking%20Generate%20Password.png?dl=0

And then, when I click Update Profile, a 3rd password field appears - with the password hidden - while the changes are being saved:

https://www.dropbox.com/s/nbyqk7cn8e9jed2/after%20clicking%20Update%20Profile.png?dl=0

I really like how Automattic implemented this in 4.3 - with you want to generate an initial password or a new one, it's the same UI: just one Generate Password button. And if you weaken the generated password, they force you to confirm that you want a weak password.

**Comment by jfarthing84** [comment]
Try it with Twenty Fifteen, or any of the default themes for that matter. Then, it _is_ how Automattic implemented it...

What theme are you using?

**Comment by jacobshare** [comment]
That's good to hear.

So it's an issue with my theme. Does it sound only CSS-related?

**Comment by jfarthing84** [comment]
I don't know. Could you try it with a default theme?

**Comment by jacobshare** [comment]
Finally got a chance to test this, and I did so with Twenty Fifteen and Twenty Sixteen:

https://www.dropbox.com/s/jovgvwmohdw45xh/tml-password-bug-2015.zip?dl=0

https://www.dropbox.com/s/49c2d5wp0f26go0/tml-password-bug-2016.zip?dl=0

In both cases, I'm seeing the exact same problems.

I tried deactivating all my plugins except for the ones necessary to continue using the membership area of the site, but that didn't change anything.

**Comment by jfarthing84** [comment]
That is very odd - that's not how it looked for me with both of those themes. You don't have a custom `profile-form.php` in either theme, do you?

**Comment by jacobshare** [comment]
No, for 2015 and 2016, I just installed and activated the vanilla themes
from WordPress.org to test. Didn't have them on the server before.

**Comment by jfarthing84** [comment]
Do you have TML's stylesheet enabled?

**Comment by jacobshare** [comment]
It was enabled. After disabling it, the problem still appears but with a
little less style, pun intended:

https://www.dropbox.com/s/rpbog8gmsxux3r8/tml-password-bug-no-css.zip?dl=0

**Comment by jfarthing84** [comment]
What browser are you using? Instead of all this back and forth, could I get credentials to test with? You can send them [here](http://www.jfarthing.com/contact).

**Comment by adamhurlebaus** [comment]
@jacobshare - I am having the same issue, but I realized that it went away when I deactivated Paid Memberships Pro.  You mentioned that you deactivated all plugins aside from your membership plugin, therefore I assume that is the same plugin you are using.  Deactivate, flush cache (if applicable), and try to load the profile page.  I imagine the issue will be resolved.

At least it worked for me so I figured I'd pass it along.

I've posted in the PMPro members section about it to see if they have a resolution.  If you have access to the forum, here is the link: http://www.paidmembershipspro.com/forums/topic/theme-my-login-issues/#post-40955

**Comment by adamhurlebaus** [comment]
Actually, here is the issue:

In Paid Membership Pro's frontend.css on lines <a href="https://github.com/strangerstudios/paid-memberships-pro/blob/dev/css/frontend.css#L180">180 - 181</a> you have this CSS: 
`/* Fixes for Twentyfiften and other themes. */
#your-profile input {display: inline-block; width: auto; }          /* front end profile via Theme My Login */`

If you remove that, everything should work as intended.

**Comment by OnGitHubSchatz** [comment]
@jacobshare I've encountered your issue in a few environments since testing the 6.4beta2 release. If you have shell/FTP access, you need to ensure the following:

1) Any earlier version is disabled and renamed from the plugin's expected path, as in '/theme-my-login-dis'
2) The 6.4beta2 release is renamed to 'theme-my-login'

Basically, even if the newer beta release is the only one active, it still inherently refers to paths with the plugin's default slug, which in this case is an earlier version. That will do a lot of ugly things without benefit of the updated AJAX and styling.

**Comment by jacobshare** [comment]
@gitschatz thanks. The only TML version installed is beta2, and I had already renamed the plugin directory

@adamhurlebaus thank you! What did it was removing those 2 css lines and re-enabling General > Enable "theme-my-login.css". Now the UI looks better and is less confusing:

https://www.dropbox.com/s/9gzcxfwbxtlt9po/tml-password-bug-resolved.zip?dl=0

@jfarthing84 I don't have a dev server running all the time, I set one up (an EC2 instance) just to test this and it's already gone, but if you want to dig around, [register on my site](https://jobmob.co.il/register) and I'll up your role
